### ISSUE TITLE
How to determine the Threshold of Detector

### ISSUE THREAD

**Issue by weiweiBC** [switchablenorms/DeepFashion2#18]
The number of samples in the results of my detector is much larger than the number of pictures. How do you set the threshold of the detector? So that neither sample loss nor redundant samples can be detected.

**Comment by geyuying** [comment]
Score thresh is 0.05 and NMS is 0.5 to balance obtaining high recall with not having too many low precision detections that will slow down inference post processing steps.

**Comment by weiweiBC** [comment]
When I set the score threshold to 0.6, the total number of samples detected was equal to the number of labels you gave (such as 30000 + pictures get 50000 + labels). Is my detector not well trained?

**Comment by geyuying** [comment]
It is normal that you detect more instances than the exact ground-truth bounding boxes. That's why we use  evaluation metric map for instance detection.

**Comment by weiweiBC** [comment]
When your score threshold is set to 0.05, will you detect a lot more samples than gt? Another question is, in your competition, if the sample detected is not in gt, will it affect the score of other samples in this picture?

**Comment by geyuying** [comment]
Yes, we will detect a lot more samples than gt. If you have questions about evaluation for retrieval competition, you can refer to issue 14.
